nifi-registry-web-api/src/main/java/org/apache/nifi/registry/NiFiRegistryApiApplication.java

```diff
@@ -54,6 +63,15 @@ protected SpringApplicationBuilder configure(SpringApplicationBuilder applicatio
                 .properties(defaultProperties);
     }
 
+    @PostConstruct
+    public void postConstruct() {
```

Comment:
I'm not sure when post construct gets called for a SpringApplicationBuilder. I don't think we want to fire the REGISTRY_START event until after the full application context is loaded, otherwise EventHookProvider consumers of that event type might not be loaded/ready yet.

Take a look at the `ApplicationListener<ApplicationReadyEvent>` interface, that could be a nice solution to this.